.github/workflows/ci_tests.yml

```diff
@@ -193,8 +206,9 @@ jobs:
   run_integrated_tests:
     needs:
       - is_not_draft_pull_request
+      - get_current_labels
       - cpu_builds
-    if: "${{ contains( fromJSON( needs.is_not_draft_pull_request.outputs.LABELS ), 'ci: run integrated tests') || github.event_name != 'pull_request' }}"
+    if: "${{ always() && (contains( fromJSON( needs.get_current_labels.outputs.LABELS ), 'ci: run integrated tests') || github.event_name != 'pull_request') }}"
```

Comment:
The addition of the `always` here is quite unsettling. What's the purpose of it?